ballerina/Client.bal

```diff
@@ -148,20 +138,15 @@ type ClientConfiguration record {|
     sql:ConnectionPool? connectionPool;
 |};
 
-# MSSQL database options.
+# Provides a set of configuration related to MSSQL database.
```

Comment:
```suggestion
# Provides a set of configurations related to MSSQL database.
```

Reply:
Since it's a set, should it be `a set of configuration`? 🤔